### ISSUE TITLE
error after adding and activating a drawing style

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#3015]
file: 
[101_W_33rd_St_Details_A.txt](https://github.com/IfcOpenShell/IfcOpenShell/files/11253583/101_W_33rd_St_Details_A.txt)

1. bpy.ops.bim.add_drawing_style()
2. bpy.ops.bim.activate_drawing_style()
3. error...

```
Python: Traceback (most recent call last):
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\blenderbim\bim\module\drawing\operator.py", line 1179, in execute
    self.set_raster_style(context)
  File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\blenderbim\bim\module\drawing\operator.py", line 1186, in set_raster_style
    style = json.loads(self.drawing_style.raster_style)
  File "C:\Program Files\Blender Foundation\Blender 3.3\3.3\python\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Blender Foundation\Blender 3.3\3.3\python\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Program Files\Blender Foundation\Blender 3.3\3.3\python\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

```

![2023-04-17_12-30-19_Blender_blender](https://user-images.githubusercontent.com/507113/232564148-18ecdbc8-dc79-42a6-8126-44d3a0bc0775.gif)

**Comment by Moult** [comment]
I haven't looked at this yet but I _think_ you need to just press "save" before activating it.

**Comment by Andrej730** [comment]
Fixed after 38cf88d5d2692a7337740c5566c26fe1bc33328f
